dashboard/test/factories/factories.rb

```diff
@@ -730,18 +730,18 @@
     script
   end
 
-  factory :stage do
-    sequence(:name) {|n| "Bogus Stage #{n}"}
+  factory :lesson, class: 'Lesson' do
```

Comment:
Does the class need to be specified (more of a question for my own understanding)?

Reply:
Oh, good catch, doesn't look like it's needed. Will remove.